config/interpolate_funcs.go

```diff
@@ -131,11 +132,16 @@ func interpolationFuncFormatList() ast.Function {
 				if !ok {
 					continue
 				}
-				parts := strings.Split(s, InterpSplitDelim)
-				if len(parts) == 1 {
+				if !IsStringList(s) {
 					continue
 				}
+
+				parts := StringList(s).Slice()
```

Comment:
This was previously relying on the fact that a bare string would have no problem being split. Need to handle it explicitly now.